### PR TITLE
the-accounts-struct.md improvement

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/src/pages/docs/the-accounts-struct.md [hunk 1]
```diff
@@ -169,8 +169,6 @@ pub struct Initialize<'info> {
 }
 ```
 
-Note the doc comment needs to be a [line or block doc comment](https://doc.rust-lang.org/reference/comments.html#doc-comments) (/// or /\*\*) to be interepreted as doc attribute by Rust. Double slash comments (//) are not interpreted as such.
```
Comment (armaniferrante) [inline docs/src/pages/docs/the-accounts-struct.md:172]: What's the motivation for this change?

Comment (stateofnonreturn) [reply docs/src/pages/docs/the-accounts-struct.md:172]: The same information is duplicated.

image
